### PR TITLE
[DOCFIX] Improve troubleshooting doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Mainly replaced the mailing list to github issues and slightly improved the debugging doc

### CODE REVIEW

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ will need to set the JVM remote debugging parameters before starting the process
 the remote debugging parameters; you can export the following configuration properties in shell or `conf/alluxio-env.sh`:
 
 ```shell
-# Java 5 through 8
```
Comment (jiacheliu3) [inline docs/en/operation/Troubleshooting.md:54]: I believe the lowest version we support is 8

File: docs/en/operation/Troubleshooting.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ After completing this setup, learn how [to attach](#to-attach).
 
 ### Debugging shell commands
 
-If you want to debug shell commands (e.g. `bin/alluxio fs ls /`), you can set the `ALLUXIO_USER_DEBUG_JAVA_OPTS` in
+If you want to debug shell commands (e.g. `bin/alluxio fs ls /`), you can set the `ALLUXIO_USER_ATTACH_OPTS` in
```
Comment (jiacheliu3) [inline docs/en/operation/Troubleshooting.md:73]: This seems like a typo

File: docs/en/operation/Troubleshooting.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/operation/Troubleshooting.md:54]: yes pls

File: docs/en/operation/Troubleshooting.md [hunk 3]
```diff
@@ -70,11 +70,11 @@ After completing this setup, learn how [to attach](#to-attach).
...
 `conf/alluxio-env.sh` as above:
 
 ```shell
-# Java 5 through 8
+# Java 8
 export ALLUXIO_USER_ATTACH_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=60000"
 # Java 9 and up
```
Comment (Xenorith) [inline docs/en/operation/Troubleshooting.md:79]: should we just call this java 11 and not have users suspect we are compatible with java 9 and up

**Review by Xenorith — CHANGES_REQUESTED** [review]

**Review by Xenorith — APPROVED** [review]

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
@Xenorith pls help cherry picking this to 2.8, thx :)

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Pull request merged**